paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-with-android/gradle.properties

```diff
@@ -1 +1 @@
-kotlin.mpp.androidSourceSetLayoutVersion1.nowarn=true
+android.useAndroidX=true
```

Comment:
this file can now be removed, since the default generated `gradle.properties` provided by `PaparazziPluginTest` will handle this.